CWE/events/Italy ahistorical.txt

```diff
@@ -0,0 +1,5248 @@
+			add_casus_belli = {
+	            target = BOS
+	            type = free_allied_cores
+	            months = 360
+            }
+			add_casus_belli = {
+	            target = MON
+	            type = free_allied_cores
+	            months = 360
+            }
+			add_casus_belli = {
+	            target = SER
+	            type = free_allied_cores
+	            months = 360
+            }
+		}
+		
+		#Italy
+		
+		#Koper, Istria
+		random_country = { 
+		    limit = { is_vassal = no war = no owns = 3345 } 
+			relation = { who = THIS value = -100 }
+			relation = { who = ITA value = -100 }
+			country_event = 8020453 
+		}
+		#Umag, Istria
+		random_country = { 
+		    limit = { tag = CRO is_vassal = no war = no owns = 3378 } 
+			3378 = { secede_province = ITA remove_core = CRO change_province_name = "Umago" }
+		}
+		#Pula, Istria
+		random_country = { 
+		    limit = { tag = CRO is_vassal = no war = no owns = 770 } 
+			770 = { secede_province = ITA remove_core = CRO change_province_name = "Umago" }
+		}
+		
+		#Croatia
+		788 = { add_core = CRO } #Mostar, Herceg Bosnia
+		2688 = { add_core = CRO } #Inta, Herceg Bosnia
+		789 = { add_core = CRO } #Livno, Herceg Bosnia
+		783 = { add_core = CRO }  #Sarajevo, Bosnia
+		784 = { add_core = CRO } #Bihac, Bosnia
+		785 = { add_core = CRO } #Banja Luka, Bosnia
+		786 = { add_core = CRO } #Tuzla, Srpska
+		787 = { add_core = CRO } #Foca, Srpska
+		2582 = { add_core = CRO } #Kotor, Montenegro
+		2538 = { add_core = CRO } #Novi Sad, Vojvodina
+		791 = { add_core = CRO } #Mitrovica, Vojvodina
+		2577 = { add_core = CRO } #Pljevlja, Sanjak
+		804 = { add_core = CRO } #Novi Pazar, Sanjak
+		
+		set_country_flag = claim_greater_croatia
+		ai_chance = { factor = 0.8 }
+	}
+
+	option = {
+		name = "Betray Italy"
+		badboy = 3 
+		
+		prestige = -50
+		
+		random_country = {
+			limit = { tag = ITA is_vassal = no }
+			relation = { who = THIS value = -150 }
+			add_casus_belli = {
+	            target = CRO
+	            type = humiliate
+	            months = 24
+            }
+		}
+		
+		ai_chance = { factor = 0.2 }
+	}
+	
+}
+
+#Muggia agreement - SLO answer
+country_event = {
+	id = 8020453
+	title = "The Koper Raid"
+	desc = EVT_8020453_DESC
+	picture = "CRO_HOS_Fighter"
+  
+	is_triggered_only = yes
+
+	option = {
+		name = "Surrender Koper"
+		prestige = -10
+		
+		any_pop = { 
+			limit = { NOT = { OR = { culture = italian culture = croat } } }
+			militancy = 2
+			consciousness = 2
+		}
+		
+		random_country = {
+			limit = { tag = ITA is_vassal = no }
+			relation = { who = THIS value = -100 }
+		}
+		random_country = {
+			limit = { tag = CRO is_vassal = no }
+			relation = { who = THIS value = -100 }
+		}
+		
+		3345 = { secede_province = ITA change_province_name = "Capodistria" }
+
+		ai_chance = { factor = 0.9 }
+	}
+
+	option = {
+		name = "They shall not have it so easily"
+		prestige = 50
+		
+		war = {
+			target = ITA
+			attacker_goal = {
+				casus_belli = acquire_core_state
```

Comment:
This should be attacker_goal = { casus_belli = acquire_core_state  state_province_id = X }
X is the first province in the state entry